lib/OpenQA/Worker.pm

```diff
@@ -750,18 +751,22 @@ sub _handle_job_status_changed ($self, $job, $event_data) {
     }
 }
 
-sub _load_avg ($field = 2) {
-    my $value = eval { (split(' ', path($ENV{OPENQA_LOAD_AVG_FILE} // '/proc/loadavg')->slurp))[$field] };
+sub _load_avg () {
+    my @load = eval { split(' ', path($ENV{OPENQA_LOAD_AVG_FILE} // '/proc/loadavg')->slurp) };
+    splice @load, 3;    # remove non-load numbers
     log_warning "Unable to determine average load: $@" if $@;
-    return looks_like_number($value) ? $value : undef;
+    return undef unless all { *looks_like_number } @load;
```

Comment:
`*looks_like_number`?
```suggestion
    return undef unless all { looks_like_number $_ } @load;
```

Reply:
```suggestion
    return () unless all { looks_like_number $_ } @load;
```

Reply:
> `*looks_like_number`?

yeah, that's https://perldoc.perl.org/perlapi#looks_like_number , seems to be working, or is this wrong?

Reply:
the function name is the right one, but not the star in front of it